diffusers/insta_flow.ipynb

```diff
@@ -0,0 +1,486 @@
+{
```

Comment:
<p>Missing some text about combining it with LoRA.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>